benchmarks/run_kurobako.py

```diff
@@ -98,7 +100,9 @@ def run(args: argparse.Namespace) -> None:
     parser.add_argument("--n-jobs", type=int, default=10)
     parser.add_argument("--sampler-list", type=str, default="RandomSampler TPESampler")
     parser.add_argument(
-        "--sampler-kwargs-list", type=str, default='{} {"multivariate":true,"constant_liar":true}'
+        "--sampler-kwargs-list",
+        type=str,
+        default='{} {\\"multivariate\\":true\,\\"constant_liar\\":true}',  # NOQA: W605
```

Comment:
```suggestion
        default=r'{} {\"multivariate\":true\,\"constant_liar\":true}',  # NOQA: W605
```

Reply:
A raw string would make it clearer to me.

Reply:
Looks nice. I updated as you suggested.